lib/graphql/relay/connection/ecto.ex

```diff
@@ -63,16 +63,17 @@ if Code.ensure_loaded?(Ecto) do
       a_after = cursor_to_offset(args[:after])
       first = args[:first]
       last = args[:last]
+      where_property = args[:where] || :id
```

Comment:
@thaterikperson Thanks again for the PR and attention to detail including cleaning up the tests. Appreciate it!

I'm still grokking this and wondering if the argument should be named something other than `where` as I'm finding it a bit confusing. Maybe `order_by`, `ordered_by`, `sort_by`, or `sorted_by`?

```elixir
Connection.Ecto.resolve(Repo, query, %{after: @a_cursor, ordered_by: :inserted_at})
```

What do you think?